### DISCUSSION TITLE
Delay when seeing type of variable

### DISCUSSION THREAD

**Discussion by Garklein** [ananthakumaran/tide#453] [Q&A] [answered]
When I put my cursor over a variable, there's a one second delay before its name and type appear in the minibuffer.
Is this just a delay with the typescript server, or can I make it show the name and type instantly?

**Comment by ananthakumaran** [comment, marked as answer]
Could you check if reducing eldoc-idle-delay value helps?

**Reply by Garklein** [reply]
That fixed it, thank you for your help!
